src/Process/SubdivisionControl.js

```diff
@@ -4,9 +4,25 @@ export default {
             (l, a) => a && a.id == layer.id && l.type == 'color');
         context.elevationLayers = context.view.getLayers(
             (l, a) => a && a.id == layer.id && l.type == 'elevation');
-    },
 
+        context.forceElevation = [];
+        for (const e of context.elevationLayers) {
+            if (e.updateStrategy.type == 1 && e.updateStrategy.options.forceLoadingStep && e.updateStrategy.options.groups) {
+                context.forceElevation = context.forceElevation.concat(e.updateStrategy.options.groups);
+            }
+        }
+        context.forceElevation.sort((a, b) => a - b);
```

Comment:
Can you comment / use a more descriptive variable name?
I think `forceElevation` is a sorted array of all defined valid level for elevation layers using the group strategy. Is that correct?